DESIGN_GUIDELINES.md

```diff
@@ -347,14 +347,14 @@ Existing Inconsistencies would not be published, that’s for our internal use 
 | Name    | Type     | Description | Notes    |
 | --- | --- | --- |--- |
 | stateMachineProps	|[`sfn.StateMachineProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-stepfunctions.StateMachineProps.html)|Optional user provided props to override the default props for `sfn.StateMachine`|
-| createCloudWatchAlarms | `boolean`|Whether to create recommended CloudWatch alarms.|
+| createCloudWatchAlarms? | `boolean`|Whether to create recommended CloudWatch alarms.|
```

Comment:
I wanted to see a default here, but looking at step-function-default.ts and lambda-stepfunctions, the default is not obvious. I'll enter a task to clarify/codify the default and update the various constructs docs.